packages/client/components/Snackbar.tsx

```diff
@@ -158,7 +152,7 @@ const Snackbar = React.memo(() => {
   }, [showSnack, transitionChildren])
 
   return portal(
-    <Modal snackbarOffset={snackbarOffset}>
+    <Modal isMeetingRoute={isMeetingRoute} isDesktop={isDesktop}>
```

Comment:
the meeting control was getting measured before it had a chance to get relocated in the DOM, which caused a bad value for snackbarOffset, which means the snackbar was placed off the screen.

also, the snackbar was lined up in the center of the screen, but now it's offset by the left nav so it's directly above the control bar

also, when navigated to/from the meeting, now the y position updates correctly